app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -455,27 +418,27 @@ public void favoritesChanged(FavoritesEvent event) {
 
     @Override
     public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
-        if (controller == null || txtvLength == null) {
+        if (controller == null) {
             return;
         }
         if (fromUser) {
             float prog = progress / ((float) seekBar.getMax());
+            int duration = controller.getDuration();
             TimeSpeedConverter converter = new TimeSpeedConverter(controller.getCurrentPlaybackSpeedMultiplier());
-            int position = converter.convert((int) (prog * controller.getDuration()));
-            txtvSeek.setText(Converter.getDurationStringLong(position));
+            int position = converter.convert((int) (prog * duration));
+            viewBinding.positionLabel.setText(Converter.getDurationStringLong(position));
+
+            if (showTimeLeft && prog != 0) {
+                int timeLeft = converter.convert(duration - (int) (prog * duration));
+                String length = "-" + Converter.getDurationStringLong(timeLeft);
+                viewBinding.durationLabel.setText(length);
+            }
         }
     }
 
     @Override
     public void onStartTrackingTouch(SeekBar seekBar) {
         // interrupt position Observer, restart later
-        cardViewSeek.setScaleX(.8f);
-        cardViewSeek.setScaleY(.8f);
-        cardViewSeek.animate()
-                .setInterpolator(new FastOutSlowInInterpolator())
-                .alpha(1f).scaleX(1f).scaleY(1f)
-                .setDuration(200)
-                .start();
```

Comment:
Something went wrong here while merging

Reply:
I think I just by mistake removed that code.

Reply:
I added the code that I think I removed. Please check whether this one is working or not.